src/main/java/uk/gov/hmcts/sptribs/controllers/CaseManagementController.java

```diff
@@ -0,0 +1,74 @@
+package uk.gov.hmcts.sptribs.controllers;
+
+import io.swagger.annotations.ApiOperation;
+import io.swagger.annotations.ApiResponse;
+import io.swagger.annotations.ApiResponses;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestHeader;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+import uk.gov.hmcts.sptribs.edgecase.event.EventEnum;
+import uk.gov.hmcts.sptribs.edgecase.model.CaseData;
+import uk.gov.hmcts.sptribs.model.CaseResponse;
+import uk.gov.hmcts.sptribs.services.CaseManagementService;
+
+@RestController
+@RequestMapping("/case/dss-orchestration")
+public class CaseManagementController {
+
+    @Autowired
+    CaseManagementService caseManagementService;
+
+    @PostMapping("/create")
+    @ApiOperation("Call CCD to create case")
+    @ApiResponses(value = {
+        @ApiResponse(code = 201, message = "created"),
+        @ApiResponse(code = 401, message = "Provided Authroization token is missing or invalid"),
+        @ApiResponse(code = 500, message = "Internal Server Error")
+    })
+    public ResponseEntity<?> createCase(@RequestHeader(HttpHeaders.AUTHORIZATION) String authorisation,
+                                        @RequestBody final CaseData caseData) {
+
+        CaseResponse createdCase = caseManagementService.createCase(authorisation, caseData);
```

Comment:
I would think we want to be adding the fields in this CaseData.java to the existing CaseData class we already have, and creating using an instance of that, with the fields added to the DssCaseData class.  Otherwise I'm not sure if all the existing Case API functionality will work with cases created through DSS.  Don't we need to have a single data object class which is used for creating both DSS cases and Case API cases?